bookkeeper-common/src/main/java/org/apache/bookkeeper/common/component/ComponentStarter.java

```diff
@@ -62,8 +62,19 @@ public void run() {
      */
     public static CompletableFuture<Void> startComponent(LifecycleComponent component) {
         CompletableFuture<Void> future = new CompletableFuture<>();
-        Runtime.getRuntime().addShutdownHook(new Thread(
-            new ComponentShutdownHook(component, future), "component-shutdown-thread"));
+        final Thread shutdownHookThread = new Thread(
+            new ComponentShutdownHook(component, future),
+            "component-shutdown-thread"
```

Comment:
I wonder if we can use a more meaning name. Not so important

Reply:
what is your suggestion?

Reply:
Something with the name/class  of the component